relay-server/src/actors/project_cache.rs

```diff
@@ -502,17 +508,13 @@ impl ProjectCacheBroker {
         }
     }
 
-    fn get_global_config(&self) -> Option<Arc<GlobalConfig>> {
+    fn global_config_value(&self) -> Option<Arc<GlobalConfig>> {
```

Comment:
```suggestion
    fn global_config(&self) -> Option<Arc<GlobalConfig>> {
```
?

Reply:
yeahh makes sense,  i went by [this](https://doc.rust-lang.org/std/ops/enum.ControlFlow.html#method.continue_value) precedent but i think it doesnt apply in the same manner here actually.